### PR TITLE
[RF] dashboardEditDescription| AT_20.02 _001

### PR DESCRIPTION
**Pull request by QAtrina**
https://trello.com/c/d6A5iBU7/943-rf-dashboardeditdescription-at2002-001

### CODE REVIEW

**Review by Julia-Cherniak — APPROVED** [review]

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 /// <reference types="cypress"/>
+import descriptionsProject from "../fixtures/descriptionsProject.json"
 
 describe(`Dashboard Edit Description`, () => {
-    const text = 'Editing description check out'
+  it('20.02 _001| Dashboard > Editing Description', () => {
+    cy.get('#description-link').contains('Add').click()
+    cy.get('.jenkins-input').type(descriptionsProject.addDescriptionProject)
+    cy.get('button[name="Submit"]').contains('Save').click()
+    cy.get('#description div:first-of-type').should('have.text', 'Project DemoQA')
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:9]: Please move `'Project DemoQA'` in the fixtures

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
 /// <reference types="cypress"/>
+import descriptionsProject from "../fixtures/descriptionsProject.json"
 
 describe(`Dashboard Edit Description`, () => {
-    const text = 'Editing description check out'
+  it('20.02 _001| Dashboard > Editing Description', () => {
+    cy.get('#description-link').contains('Add').click()
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:6]: Please move `'Add'` in the fixtures

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
+    cy.get('.jenkins-input').type(descriptionsProject.addDescriptionProject)
+    cy.get('button[name="Submit"]').contains('Save').click()
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:8]: Please move `Save` in the fixtures

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
 
-    it(`Dashboard > Editing Description`, () => {
-        cy.get('#description-link').click()
-        cy.get('.jenkins-input')
-          .clear()
-          .type(text)
-        cy.get('button[name="Submit"]').contains('Save').click()
-        cy.get('#view-message').should('contain', text)
-    });
+    cy.get('#description-link').contains('Edit').click()
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:11]: Please move `Edit` in the fixtures

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
+    cy.get('.jenkins-input').clear().type(descriptionsProject.editDescriptionProject)
+    cy.get('button[name="Submit"]').contains('Save').click()
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:13]: Please move `Save` in the fixtures

File: cypress/e2e/dashboardEditDescription.cy.js [hunk 6]
```diff
@@ -1,16 +1,18 @@
...
+    cy.get('#description div:first-of-type').should('have.text',' - QA JS 06')
```
Comment (Rom-T) [inline cypress/e2e/dashboardEditDescription.cy.js:14]: Please move `' - QA JS 06'` in the fixtures

**Review by MariaMoroz — APPROVED** [review]

**Pull request merged**